documentation/Docker-DNS.md

```diff
@@ -0,0 +1,86 @@
+# Docker DNS setup
+
+There are several DNS components in the Docker setup:
+
+* A non-validating resolver, used for DNS resolving by almost all tests.
+  As we have our own DNSSEC validation test, we want to see bogus responses as well.
+* A validating resolver, used to validate DANE records through ldns-dane.
+* An authoritative server for the connection test zone.
+* The DNS records in the zone for `INTERNETNL_DOMAINNAME`.
+  These are hosted elsewhere, but have certain requirements for the instance to work.
+
+The resolvers do not require any specific configuration.
+In this document, `INTERNETNL_DOMAINNAME` is `example.com`.
+
+## Common parts
+
+For accessing the absolute minimum basic functionality of the application the following DNS records must be configured:
+
+    example.com.                    A      192.0.2.1
+                                   AAAA   2001:db8:1::1
+
+The following extra records must be configured for language switching:
+
+    www.example.com.                CNAME  example.com.
```

Comment:
The www is not really needed for language switching right?
It's there as fallback (it has to be configured because of the certificate request for it of course). Maybe move it to the apex block above.

Reply:
I think the www record should just always exist. Will move it up. And will check certbot config against this new document.